decimal.go

```diff
@@ -279,20 +284,23 @@ func (d Decimal) Cmp(d2 Decimal) int {
 	return rd.value.Cmp(rd2.value)
 }
 
+// Equals returns whether d == d2 at the smallest scale among d and d2.
```

Comment:
This comment is confusing, just say that it returns whether the numbers represented by d and d2 are equal, regardless of their internal decimal representation.